cogs/fun.py

```diff
@@ -178,17 +180,17 @@ def reaction_check(m):
         try:
             await msg.add_reaction("🍻")
             await self.bot.wait_for("raw_reaction_add", timeout=30.0, check=reaction_check)
-            await msg.edit(content=f"**{user.name}** and **{ctx.user.name}** are enjoying a lovely beer together 🍻")
+            await ctx.followup.edit_message(message_id=msg.id,content=f"**{user.name}** and **{ctx.user.name}** are enjoying a lovely beer together 🍻")
```

Comment:
Previous implementation works just fine here since followups always return the WebhookMessage. The method used here is technically not for interactions either.